### ISSUE TITLE
not finding libxml2?

### ISSUE THREAD

**Issue by dcollinsLMC** [leedm777/homebrew-asterisk#23]
I have installed libxml2 via homebrew (brew install libxml2).  I am still getting this message:

Last 15 lines from /Users/david/Library/Logs/Homebrew/asterisk/01.configure:
checking curses.h presence... yes
checking for curses.h... yes
checking for initscr in -lncurses... yes
checking for curses.h... (cached) yes
checking for uuid_generate_random in -luuid... no
checking for uuid_generate_random in -le2fs-uuid... no
checking for uuid_generate_random... yes
checking for json_dumps in -ljansson... yes
checking jansson.h usability... yes
checking jansson.h presence... yes
checking for jansson.h... yes
checking for clock_gettime in -lrt... no
checking for xml2-config... /usr/bin/xml2-config
configure: *** The Asterisk menuselect tool requires the 'libxml2' development package.
configure: *** Please install the 'libxml2' development package.

**Comment by leedm777** [comment]
@dcollinsLMC homebrew-asterisk uses the system installed OS X, instead of the version installed by Homebrew. What versions of Xcode and macOS/OS X are you running on? Did you install the Xcode command line tools (`xcode-select --install`)?

**Comment by dcollinsLMC** [comment]
Yes I have run xcode-select --install.  I am running Mac OS X 10.11.6 and Xcode 8.0.  Just tried it again after running brew uninstall libxml2, same result.

**Comment by dcollinsLMC** [comment]
Well, I upgraded to 10.12 (Sierra), did xcode-select --install, and it now builds.  Hooray!

**Comment by leedm777** [comment]
Yay! 👏🏻

**Comment by leedm777** [comment]
Just ran into this myself.

Appears to be an incompatibility between Xcode 8 and Mac OS X 10.11 (or CLT 7.3.1). Looks like the solution is to a) upgrade to Sierra b) downgrade to Xcode 7 c) change homebrew-asterisk to use the homebrew version of libxml2.

I [may be stuck on El Capitan](https://youtrack.jetbrains.com/issue/IDEA-158500) for some time, so I might try going down path c.

**Comment by leedm777** [comment]
Whelp. Looks like Asterisk does some [path](https://github.com/asterisk/asterisk/blob/7126194187f1ddae1a6e7a7f80f5e84bafd6db4b/autoconf/ast_ext_tool_check.m4#L15) [munging](https://github.com/asterisk/asterisk/blob/7126194187f1ddae1a6e7a7f80f5e84bafd6db4b/autoconf/ast_ext_tool_check.m4#L19) to work better with cross-compiling. It's not immediately clear how to fix it for homebrew and not break cross compiling.

😞
